flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceFactoryTest.java

```diff
@@ -79,6 +80,49 @@ public void testNoMatchedTable() {
                 .hasMessageContaining("Cannot find any table by the option 'tables' = " + tables);
     }
 
+    @Test
+    public void testExcludeTable() {
+        inventoryDatabase.createAndInitialize();
+        Map<String, String> options = new HashMap<>();
+        options.put(HOSTNAME.key(), MYSQL_CONTAINER.getHost());
+        options.put(PORT.key(), String.valueOf(MYSQL_CONTAINER.getDatabasePort()));
+        options.put(USERNAME.key(), TEST_USER);
+        options.put(PASSWORD.key(), TEST_PASSWORD);
+        // db has three tables , table.list= (products,orders shipments)
+        options.put(TABLES.key(), inventoryDatabase.getDatabaseName() + ".prod\\.*");
+        String tableExcludeList = inventoryDatabase.getDatabaseName() + ".prod\\.orders";
```

Comment:
Here the inclusion pattern is `prod.*`, which matches only one table `products`. The exclusion pattern is `prod.orders`, which doesn't match any table, so I'm afraid this case doesn't truly test if the exclusion works as expected. 

What about using `\\.*` for matching all tables in inclusion, and `orders` for exclusion? The expected output should be `products` and `shipments` after applying the filter.

Reply:
u are right and  I will adapter it，tks。